ublox/src/ubx_packets/packets.rs

```diff
@@ -1974,17 +1939,25 @@ struct RxmRtcm {
 #[ubx(class = 0x10, id = 0x02, max_payload_len = 1240)]
 struct EsfMeas {
     time_tag: u32,
-    #[ubx(map_type = EsfMeasInfo, from = EsfMeasInfo::new)]
+    #[ubx(
+        map_type = EsfMeasInfo,
+        from = EsfMeasInfo::new,
+        flatten,
+        flat_fields = ["flags", "u16", "id", "u16", "data", "U32Iter"]
```

Comment:
@notoriaga 

like here... made it so we can provide which fields to bubble up but we have to provide the types for them... 
theres also a thing im not sure how to fix to translate 
`format_ident!("Option<u32>")` looks like throws an error...

Reply:
can do some really hacky thing...
check if its str Option, then just wrap it if else -1